### PR TITLE
Speaking page redesign

### LINKED ISSUE #9: Redesign speaking page
It looks OK but isn't great – especially the big list of talks with italics text – very difficult to read.

### PR DESCRIPTION
**Pull request by ashfurrow**
Deployed to http://staging.ashfurrow.com/speaking/

Fixes #9.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Als I'm pretty sure that I'm doing this in, like, the worst way possible ¯\\_(ツ)_/¯ Web developers look away.

File: source/speaking.html.erb [hunk 1]
```diff
@@ -7,70 +7,165 @@ og_image: /img/speaking.jpg
...
+    html += "<p><a href='#{code}'><i class='fa fa-youtube-play'></i>Recorded Video</a></p>"
   end
 
-  if code
-    html += " <a href='#{ code }'><i class='fa fa-code'></i></a>"
-  end
+  html
+end
 
+def columns(talks)
```
Comment (ashfurrow) [inline source/speaking.html.erb]: Writing this function in particular made me feel bad.

**Comment by orta** [comment]
I think you need a thumbnail of the first slide for all of those past ones, sorry, it's effort I know, but it just feels really boring reading through that section. Too repetitive. Rough idea:

screen shot 2015-05-03 at 23 53 22

**Comment by orta** [comment]
But in general, I love this, I have to build something like it.

**Comment by ashfurrow** [comment]
Thanks – I agree, it's too repetitive. Definitely something involving the slides' imagery would help. Not exactly a time-sensitive problem; this has been on my TODO list since last Summer when I started the project, so I don't mind delaying it.

**Comment by ashfurrow** [comment]
@orta thoughts on something like this? 

screen shot 2015-05-07 at 11 34 09 am

Obviously pretty rough, but you get the idea.

**Comment by orta** [comment]
rough feel: Might make the whole thing too long, or too big, maybe 3-4 on a row might make it feel nicer, it'd end up extremely image heavy with this pattern.

**Comment by ashfurrow** [comment]
Yeah, good points. I'll let it sit for a few more days :)

**Comment by ashfurrow** [comment]
@AshleyKrista made this suggestion:

screen shot 2015-05-07 at 11 58 21 am

Obviously still pretty image/design-heavy, but I'm actually kind of in love with it.

**Comment by orta** [comment]
Love the idea, could be worth trying aligning the text to the right too. Not sold on one column, could try it at 2/3rds of current size with no margins horizontal/vertical too

**Comment by AshleyKrista** [comment]
I think two columns for sure. 

tricky tricky

**Comment by ashfurrow** [comment]
Either wider + two columns or narrower + one column.

**Comment by AshleyKrista** [comment]
Wider, one column, and combine Orta's more interesting text layout with what you have! (Just make the text bigger and the conf name biggest)

**Comment by ashfurrow** [comment]
Ah, so use the conference name as a header? That might make more sense – hadn't realized what kind of grounding it had lost without a header.

**Comment by ashfurrow** [comment]
So the latest changes are deployed to http://staging.ashfurrow.com/speaking/ . All that conversation and I pretty much came back to Orta's suggestion: 

screen shot 2015-05-08 at 10 40 02 pm

Also switched to hiding some of the repeated talks; having them there once is enough, and reduces the page length.

**Comment by ashfurrow** [comment]
Things don't flow _great_ on mobile, but I'm willing to live with it at this point. Maybe touch-up later.

**Comment by ashfurrow** [comment]
From esteemed designer friend of mine:

> Hey, looking good. It's a little weird that "upcoming" is two columns and "already spoke" is just one. Maybe put conf logos in the 'upcoming' area where your decks are now?

**Comment by orta** [comment]
you could  make the opening slides for your upcoming talks?

**Comment by ashfurrow** [comment]
Would be a good idea if I had my slides prepared that far in advance :P

**Comment by orta** [comment]
You don't need your slides, just a title :P

**Comment by ashfurrow** [comment]
Fair enough! I sometimes don't even know that, though :smile_cat: I think I'll play around and try both.

**Comment by ashfurrow** [comment]
Yeah, I was thinking about it, and the upcoming talks section is for people to see what conferences I'm at, not what I'm talking about. Went with the logos. Thanks everyone for your help!

**Pull request merged**